test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -124,10 +124,15 @@ public void TestBasicClassWithNullCharactor()
 ";
```

Comment:
this is invalid?

Reply:
During Yaml de-serialization, we treat scalar value ~ as null, therefore it will be removed during the null value validation

---
In reply to: [200322112](https://github.com/dotnet/docfx/pull/2994#discussion_r200322112) [](ancestors = 200322112)